packages/zefyr/pubspec.yaml

```diff
@@ -13,7 +13,11 @@ dependencies:
   collection: ^1.14.6
   url_launcher: ^5.0.0
   quill_delta: ^1.0.0
-  notus: ^0.1.0
+  notus:
+    git:
+      url: https://github.com/kekland/zefyr.git
```

Comment:
s/kekland/shinyinc/ ?

Reply:
oh, forgot to change that in the zefyr repo, will change it with changes to numbered list rendering